spec/user_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'user'
+
+describe User do
+  let(:diego) { User.new '+44 7496548640' }
+  let(:diego_fake) {double(:diego_fake, :place_food_order => true, :food_menu_food_order => {} ) }
+  let(:list) { double(:list) }
+  let(:place_food_order) { double(:place_food_order) }
+
+
+  it 'has a phone number' do
+    expect(diego.phone).to_not be_empty
+  end
+
+  describe '#show_menu' do
+    it 'shows Default-Dishes' do
+      expect(diego.show_menu).to have_at_least(2).dishes
+    end
+    it 'corresponds to #available_menu' do
+      expect(diego.show_menu).to eq diego.food_menu.available
+    end
+  end
+
+  describe '#select_food_to_order' do
+    it 'creates current #food_order' do
+      diego.show_menu
+      expect(diego.select_food_to_order(:risotto, :risotto)).to eq({:risotto => 2})
```

Comment:
Line is too long. [83/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.